.gitignore

```diff
@@ -33,3 +33,7 @@ UserInterfaceState.xcuserstate
 # Test coverage
 .nyc_output/
 coverage/
+
+# ignore lock files
+package-lock.json
```

Comment:
the lock files are not supposed to be ignored. They are used to determine the exact packages versions